tasks/updateconstants.js

```diff
@@ -224,3 +228,11 @@ function expandItemGroup(key, items) {
     return base;
   }
 }
+
+function replaceRepeatedNumbersWithSlashes(strToReplace) {
+  return strToReplace.replace(/(\d{1,3})\/(\1\/){1,2}\1/, '$1');
```

Comment:
It only catches 3 to 4 groups of repeated numbers of length 1 to 3, separated with slashes, so it's hard to match anything that we don't want to replace, and before the call I assure that the field exist.

I can't see it breaking, but I thougth in some limitations. Now it doesn't fix multiple times in the same object, and doesn't catch if there are spaces between the numbers and slashes.